src/components/tiles/table/use-tile-api.ts

```diff
@@ -6,18 +6,21 @@ import { ITileApi } from "../tile-api";
 
 interface IProps {
   content: TableContentModelType;
-  getTitle: () => string | undefined;
   getContentHeight: () => number | undefined;
   exportContentAsTileJson: () => string;
   onRegisterTileApi: (tileApi: ITileApi, facet?: string | undefined) => void;
   onUnregisterTileApi: (facet?: string | undefined) => void;
 }
 export const useToolApi = ({
-  content, getTitle, getContentHeight, exportContentAsTileJson, onRegisterTileApi, onUnregisterTileApi
+  content, getContentHeight, exportContentAsTileJson, onRegisterTileApi, onUnregisterTileApi
 }: IProps) => {
   const contentRef = useCurrent(content);
   const tileApi: ITileApi = useMemo(() => ({
-    getTitle,
+    // TODO: we should be able to remove getTitle from the tool api and instead
+    // all tile content models can provide a title property. In some cases this is a
```

Comment:
This probably isn't the time to worry about it, but the general pattern is for the tile title to live in the wrapper model rather than the content. I feel like the preference would be to access the title through the wrapper, even though that might not always be easy to do. But in the general case, accessing the title from the content will require some `getParent` gymnastics that aren't ideal.

Reply:
Oh right, I forgot about the wrapper with the title.  I'll update the comment.